README.md

```diff
@@ -13,41 +13,14 @@ encourage more programmers to become a part of the Haskell community.
 
 ## Prerequisites (what you need to have locally)
 
-For the project to build, you will need to have `libpq-dev` installed on you computer. You can install it by running the command `sudo apt install libpq-dev`.
+You will need to have the following installed on your system in order to build and test `issue-wanted`. Click on the links to learn how to install each one:
 
-You will also need to setup Postgres on your computer. Here are the instructions for doing so:
+1. [ghc](https://downloads.haskell.org/~ghc/6.8.3/docs/html/users_guide/installing-bin-distrib.html)
```

Comment:
Wow, `6.8.3` is a very old GHC version. Let's link to `ghcup` instead:

* https://www.haskell.org/ghcup/

Reply:
Yes, good call. Thank you!